hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/HoodieInlineTestSuiteWriter.java

```diff
@@ -65,7 +65,14 @@ public HoodieInlineTestSuiteWriter(JavaSparkContext jsc, Properties props, Hoodi
   }
 
   public void shutdownResources() {
-    // no-op for non continuous mode test suite writer.
+    if (cfg.useDeltaStreamer) {
+      log.info("Shutting down DS wrapper gracefully ");
+      this.deltaStreamerWrapper.shutdownGracefully();
```

Comment:
if deltastreamer wrapper is not null, we can just shut it down? as in we should not need to check the config to do this